app/views/games/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Available Games</h1>
+
+<% @game.each do |game| %>
```

Comment:
This looks good. Could make it simpler.
```
<table>
  <% @games.each do |game| %>
    <tr><%= game.name %></tr>
  <% end %>
</table>
```

- adds a table to make it display a bit nicer.
- `game` will always be present in this context because active record won't return a `nil` game from `Game.available`